tasks/configure.yml

```diff
@@ -0,0 +1,56 @@
+---
+# ss-configure
+
+# Create api-key if not defined
+- set_fact: archivematica_src_configure_ss_api_key={{ 999999999999999999999 | random | to_uuid | hash('md5') }}
+  when: "archivematica_src_configure_ss_api_key is undefined and archivematica_src_configure_ss|bool"
+
+# Delete default SS superuser test
+- name: "Delete default SS superuser"
+  become: "yes"
+  shell: |
+    cd {{ archviematica_src_ss_app }}
```

Comment:
I think that using `cd`, and `chdir` later, is not needed

Reply:
Yes, this "cd" is not needed. The typo is prior to this change, and this variable is used several times in the role. I'll include the typo fix as a separate commit.